### ISSUE TITLE
Luminosity ratio for the keyboard focus indicator of buttons under 'Add action' blade is less than required.

### ISSUE THREAD

**Issue by hartra344** [Azure/LogicAppsUX#4038]
### Describe the issue

https://msazure.visualstudio.com/One/_workitems/edit/26531803 

User Experience:  
Low vision will find it difficult in identifying which control the current focus indicator is on if the focus indicator has insufficient color contrast. As a result, the user will face difficulty in performing the desired action related to that control and may skip the functionality of that control or element while navigating through the keyboard.
Note: User credentials should NOT be included in the bug.
Pre-requisites:
Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed.
Navigate to Azure control present on the right side and select it.
Navigate to LogicApp icon present under workspace section, click on that and select 'Create New Project'.
 Navigate select stateful workflow, then enter the name of the workflow.
Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter.
Navigate to 'Open in current window' and select it.
A dialog will appear, select 'Yes', now the workflow will open in new window.
Repro Steps: 
Open Visual Studio app.
Navigate to any workflow and right click on it, then select 'open designer'.
Navigate to '+' button and select 'Add Action' control.
Navigate all the controls of 'Add Action' blade.
Observe whether Luminosity ratio for the keyboard focus indicator of buttons under 'Add action' blade or not.
Actual Result: 
Luminosity ratio for the keyboard focus indicator of buttons under 'Add action' blade is 2.4:1, which is less than the minimum required ratio i.e., 3:1.
Expected result: 
Luminosity ratio for the keyboard focus indicator of buttons under 'Add action' blade should be equal to or more than 3:1.
ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance/previous test pass bug id's: 
N/A 
UIA/DOM details available: 
N/A
MAS Reference: 
[MAS 1.4.11 – Non-text Contrast](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7bB19D1152-56E4-44F4-832A-A650AC312C4E%7d&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true)
Reference Links: 
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”. 
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

IN ADO Item

### Browser

Chrome
